reana_db/models.py

```diff
@@ -182,7 +182,7 @@ class Job(Base, Timestamp):
     max_restart_count = Column(Integer)
     deleted = Column(Boolean)
     logs = Column(String, nullable=True)
-    prettified_cmd = Column(String(1000))
+    prettified_cmd = Column(Text)
```

Comment:
Should we change `cmd` to `Text` as well?

Reply:
I think you are right, at first I thought that there wouldn't be any problem because variables are not expanded, but taking into account [how `scripts` can be defined in `yadage`](https://yadage.readthedocs.io/en/latest/definingworkflows.html?highlight=script#process-definitions) we better change it as well.